### PR TITLE
MD docs update to support Docusaurus v3 upgrade

### PR DESCRIPTION
**Pull request by bluvulture**
## Changes in this pull request  
Resolves #

### <samp>🤖 Generated by Copilot at a36fe62</samp>

This pull request fixes various formatting issues in the documentation pages, by escaping special characters that interfere with Markdown or HTML rendering. It also adds some links to relevant resources and folders in the documentation pages, to improve their accessibility and consistency.

### <samp>🤖 Generated by Copilot at a36fe62</samp>

> _Oh we are the docs team and we work with Markdown_
> _We fix the formatting and we make the links sound_
> _We escape the `<>` and the `{}` too_
> _We heave and we ho and we pull the changes through_

### <samp>🤖 Generated by Copilot at a36fe62</samp>

*  Escape curly braces, less than and greater than signs in Markdown tables and notes to fix rendering issues ([link](https://github.com/pimcore/pimcore/pull/16262/files?diff=unified&w=0#diff-d5c073f5ef70311bcf27b171e6f8d28f54ebb1c2be92d22eecd7ac824c32bcc6L116-R116), [link](https://github.com/pimcore/pimcore/pull/16262/files?diff=unified&w=0#diff-88f9e769e57fa42fef09f83669a719375e1da9e686fcfa426b136202d4ca9c6aL95-R95), [link](https://github.com/pimcore/pimcore/pull/16262/files?diff=unified&w=0#diff-7c063b3fbacae397551206f70eafe616ce203d2492240702041c4c7f2d161c6dL73-R73), [link](https://github.com/pimcore/pimcore/pull/16262/files?diff=unified&w=0#diff-870f089536a0d99cdddcbe103e2a521d42e4521e94468f8badd3c3ca2d7ea528L47-R47), [link](https://github.com/pimcore/pimcore/pull/16262/files?diff=unified&w=0#diff-ff8710cffe01f4b10a40fbc44fd43d131c90ef202b329214a4877a9fdc2451d2L12-R12), [link](https://github.com/pimcore/pimcore/pull/16262/files?diff=unified&w=0#diff-747f9f12e05a44ab4a676e5945fcac1161be8997046949173ee154e95fb6d270L19-R19), [link](https://github.com/pimcore/pimcore/pull/16262/files?diff=unified&w=0#diff-884ecc0e4b291532bfeb2aedd09660b869076bd0787d66cc3663a90e43bb02eaL4-R8), [link](https://github.com/pimcore/pimcore/pull/16262/files?diff=unified&w=0#diff-884ecc0e4b291532bfeb2aedd09660b869076bd0787d66cc3663a90e43bb02eaL166-R169), [link](https://github.com/pimcore/pimcore/pull/16262/files?diff=unified&w=0#diff-375aded12dfb868b84d5b8c27d1aed2358680a27b683e8a985f1b85193dd79aaL177-R177))
* Add links to frontend and admin interface URLs in Docker installation guide for convenience and consistency ([link](https://github.com/pimcore/pimcore/pull/16262/files?diff=unified&w=0#diff-46b5bdf332a66cc8a785f041eb629f5e4e39e539fa8e4f0c84586ffcc2c72734L46-R48))
* Add links to Symfony documentation, .gitignore file, and Pimcore configs folders in various guides for accessibility and consistency ([link](https://github.com/pimcore/pimcore/pull/16262/files?diff=unified&w=0#diff-94554b8a0a1feffa53be86512c5132b894cacee04512e4a8b91df1e07368ad96L15-R15), [link](https://github.com/pimcore/pimcore/pull/16262/files?diff=unified&w=0#diff-22d2f895a3f3d816034d6be79f2ae306833d2c3dccdbf930da185d6989099fa4L7-R7), [link](https://github.com/pimcore/pimcore/pull/16262/files?diff=unified&w=0#diff-d39e3f0461133fdc79416157bdcc5f5f98cacecae0a12b90f1385cdaf1ca8855L16-R17), [link](https://github.com/pimcore/pimcore/pull/16262/files?diff=unified&w=0#diff-b0cf9bd66612826b7114395f825930bef3d9429bf92204da63df4ea8df3053a9L12-R14))
* Escape less than and greater than signs in HTML sanitizer configuration to fix formatting ([link](https://github.com/pimcore/pimcore/pull/16262/files?diff=unified&w=0#diff-c601cb407d4defaaf4f0ab272abee11dcc09f7984d5f296e824f86e72ed7e46aL68-R68))

### CODE REVIEW

File: doc/02_MVC/04_Routing_and_URLs/02_Custom_Routes.md [hunk 1]
```diff
@@ -113,7 +113,7 @@ to do so just embrace the part with curly brackets { } (see example below).
 
 | Name     | Pattern                | Reverse          | Controller                                    | Variables | Defaults     | Site IDs | Priority     | Methods     |
 |----------|------------------------|------------------|-----------------------------------------------|-----------|--------------|----------|--------------|-------------|
-| news category  | /\\/news-category\\/([^_]+)_([0-9]+)(_category_)?([0-9]+)?/ |  /news-category/%text_%id{_category_%categoryId}    | App\Controller\NewsController::listingAction  | text,id,text2,categoryId   |              |        | 1              |       |
+| news category  | /\\/news-category\\/([^_]+)_([0-9]+)(_category_)?([0-9]+)?/ |  /news-category/%text_%id\{_category_%categoryId}\    | App\Controller\NewsController::listingAction  | text,id,text2,categoryId   |              |        | 1              |       |
```
Comment (jdreesen) [inline doc/02_MVC/04_Routing_and_URLs/02_Custom_Routes.md]:
```suggestion
| news category  | /\\/news-category\\/([^_]+)_([0-9]+)(_category_)?([0-9]+)?/ |  /news-category/%text_%id\{_category_%categoryId\}    | App\Controller\NewsController::listingAction  | text,id,text2,categoryId   |              |        | 1              |       |
```

File: doc/03_Documents/01_Editables/40_WYSIWYG.md [hunk 2]
```diff
@@ -65,5 +65,5 @@ framework:
 ```
 If you want to adapt this configuration please have a look at the [symfony documentation](https://symfony.com/doc/current/html_sanitizer.html). Add your custom configuration to you project, e.g. to `config/packages/html_sanitizer.yaml`
 
-> Note: When using API to set WYSIWYG data, please pass encoded characters for html entities e.g. <,>, & etc.
+> Note: When using API to set WYSIWYG data, please pass encoded characters for html entities e.g. `<,>`, & etc.
```
Comment (jdreesen) [inline doc/03_Documents/01_Editables/40_WYSIWYG.md]:
```suggestion
> Note: When using API to set WYSIWYG data, please pass encoded characters for html entities e.g. `<`, `>`, `&`, etc.
```

File: doc/23_Installation_and_Upgrade/09_Upgrade_Notes/README.md [hunk 3]
```diff
@@ -174,7 +174,7 @@ The tokens for password reset are now stored in the DB and are one time use only
 
 -  Enabled Content Security Policy by default.
 -  Implemented Symfony HTML sanitizer for WYSIWYG editors. Please make sure to sanitize your persisted data with help of this [script](https://gist.github.com/dvesh3/0e585a16dfbf546bc17a9eef1c5640b3).
-Also, when using API to set WYSIWYG data, please pass encoded characters for html entities <,>, & etc.
+Also, when using API to set WYSIWYG data, please pass encoded characters for html entities `<,>`, & etc.
```
Comment (jdreesen) [inline doc/23_Installation_and_Upgrade/09_Upgrade_Notes/README.md]:
```suggestion
Also, when using API to set WYSIWYG data, please pass encoded characters for html entities `<`, `>`, `&`, etc.
```

**Review by fashxp — CHANGES_REQUESTED** [review]
please rebase once more to 11.1 and force-push to that branch in order to make sure there are not unrelated commits in that PR.

**Review by fashxp — APPROVED** [review]

**Pull request merged**